metaflow/plugins/pypi/micromamba.py

```diff
@@ -173,10 +203,15 @@ def create(self, id_, packages, python, platform):
         for package in packages:
             cmd.append("{url}".format(**package))
         self._call(cmd, env)
+        self.logger(
+            f"Created {platform} virtual environment {id_} in {time.time() - start_time:.2f}s!"
+        )
 
+    @functools.lru_cache(maxsize=None)
     def info(self):
         return self._call(["config", "list", "-a"])
 
+    @functools.lru_cache(maxsize=None)
```

Comment:
to check if there is any unintended consequence of this

Reply:
There is an implicit `None` returned from this call which will be cached, if the environment has not been formed yet. This might cause problems in some cases